src/app/components/transcriptions/approve-transcript/approve-transcript.component.ts

```diff
@@ -0,0 +1,69 @@
+import { CommonModule, DatePipe } from '@angular/common';
+import { Component, OnInit, inject } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { DetailsTableComponent } from '@common/details-table/details-table.component';
+import { GovukHeadingComponent } from '@common/govuk-heading/govuk-heading.component';
+import { ReportingRestrictionComponent } from '@common/reporting-restriction/reporting-restriction.component';
+import { TranscriptionDetails } from '@darts-types/transcription-details.interface';
+import { HeaderService } from '@services/header/header.service';
+import { TranscriptionService } from '@services/transcription/transcription.service';
+import { ViewTranscriptComponent } from '../view-transcript/view-transcript.component';
+import { ApproveTranscriptButtonsComponent } from './approve-transcript-buttons/approve-transcript-buttons.component';
+
+@Component({
+  selector: 'app-approve-transcript',
+  standalone: true,
+  imports: [
+    CommonModule,
+    ViewTranscriptComponent,
+    ApproveTranscriptButtonsComponent,
+    DetailsTableComponent,
+    GovukHeadingComponent,
+    ReportingRestrictionComponent,
+  ],
+  templateUrl: './approve-transcript.component.html',
+  styleUrl: './approve-transcript.component.scss',
+})
+export class ApproveTranscriptComponent implements OnInit {
+  headerService = inject(HeaderService);
+  route = inject(ActivatedRoute);
+  transcriptionService = inject(TranscriptionService);
+  datePipe = inject(DatePipe);
+
+  caseDetails = {};
+  hearingDetails = {};
+  reportingRestriction: string | null = null;
+
+  transcriptId = this.route.snapshot.params.transcriptId;
+  transcript$ = this.transcriptionService
```

Comment:
The $ postfix is a convention for observables. As the observable is subscribed to a subscription is returned.

Instead, we should probably subscribe to transcript$ with async pipe.

The mapping logic can be done like:

` vm$ = this.transcriptionService .getTranscriptionDetails(this.transcriptId).pipe(map((data: TranscriptionDetails)=>{
    return {
      reportingRestriction: ...
      caseDetails: ...
      hearingDetails: ...
}
}))`

So in the template we get a vm$ | async as vm. We get vm.caseDetails etc